Automattic/wp-calypso
# Related posts: update demo images with compressed versions

#### Changes proposed in this Pull Request

* Related posts: update demo images with compressed versions.
  * Replaces two of the existing images for compressed versions.

#### Notes

* Initial PR: https://github.com/Automattic/wp-calypso/pull/31518
* Compressed images from Jetpack PR: https://github.com/Automattic/jetpack/pull/11623

#### Testing instructions

* Spin up this PR.
* Go to `/settings/traffic/SITE_URL`.
* Make sure you see the correct assets in the Related Posts module configuration.
